snooty/rstspec.toml

```diff
@@ -886,7 +886,14 @@ options.type = "string"
 [directive.video]
 help = "Embed a video in your article"
 argument_type = "uri"
-example = """.. video:: ${1: full video url}"""
+options.title = "string"
```

Comment:
do we need to warn for partial options. in the case that they put only title and [wondering why SD does not show up](https://github.com/mongodb/snooty/pull/1236/files#diff-3f8a5baf91118745fc1b50041ac882a0ce473b5aab68c615832d9aad05a0ac5bR77)

Reply:
I originally wanted to keep usage of the options more loose in case we wanted to use them for non-structured data things in the future. That being said, we're only using them for structured data for now, so I went ahead and added some logic to warn against this.

Just noting here that even though `description` is optional for Google, I made it "required" in the parser to encourage usage of it along with the other options and to simplify the logic. Let me know if this works for you or if you need me to change anything.